src/chapters/http/exercises.rst

```diff
@@ -0,0 +1,190 @@
+Exercises: HTTP In The Browser
```

Comment:
I like this section a lot. Lots of good, approachable practice for the students.

Reply:
Thanks!